requirements.txt

```diff
@@ -178,7 +178,7 @@ open-dread-rando==2.9.0
     # via randovania (setup.py)
 open-prime-rando==0.12.0
     # via randovania (setup.py)
-open-samus-returns-rando==1.1.1
+open-samus-returns-rando==1.2.0
```

Comment:
Since this is our version we're planning to use for stable, lets bump the version in pyproject.toml too

Reply:
Done